operator/pkg/images/images.go

```diff
@@ -199,7 +216,27 @@ func GetCassandraImage(serverType, version string) (string, error) {
 	}
 
 	if !found {
-		return "", fmt.Errorf("server '%s' and version '%s' do not work together", serverType, version)
+		// For fallback images, just return the image name directly
+		fallbackImageName := ""
+
+		if serverType == "dse" {
+			if !IsDseVersionSupported(version) {
+				return "", fmt.Errorf("server 'dse' and version '%s' do not work together", version)
+			}
+			fallbackImageName = fmt.Sprintf("datastax/dse-server:%s", version)
+		} else {
+			if !IsOssVersionSupported(version) {
+				return "", fmt.Errorf("server 'cassandra' and version '%s' do not work together", version)
+			}
+			versionWithUnderscores := strings.Replace(version, ".", "_", -1)
+			fallbackImageName = fmt.Sprintf("datastax/cassandra-mgmtapi-%s:v0.0.1", versionWithUnderscores)
```

Comment:
do we fix the 0.0.1 somewhere else?

Reply:
This has now been removed because of the newer naming scheme.